src/masternodes/mn_checks.h

```diff
@@ -313,11 +313,13 @@ struct CUpdateTokenMessage {
 
 struct CMintTokensMessage : public CBalances {
     using CBalances::CBalances;
+    CScript to;
 
     ADD_SERIALIZE_METHODS;
     template <typename Stream, typename Operation>
     inline void SerializationOp(Stream &s, Operation ser_action) {
         READWRITEAS(CBalances, *this);
+        READWRITE(to);
```

Comment:
Cannot just change serialisation, this will break all previous mint token messages as to will always be expected. See this link below for optional serialisation.

https://github.com/DeFiCh/ain/blob/221a8489a632098f1dbb1cc1af84de98aba369c6/src/masternodes/mn_checks.h#L251-L254

Reply:
Fixed: [174f993](https://github.com/DeFiCh/ain/pull/1650/commits/174f9932dc7a291626cc132673d64e11700936f6)